docs/guides/tracing.md

```diff
@@ -187,6 +187,82 @@ CREATE TABLE system_traces.node_slow_log (
 * `table_names`: a list of tables used for this query, where applicable
 * `username`: a user name used for authentication with this query
 
+### Lightweight (fast) slow queries logging mode
+
+#### The motivation
+
+Natural desire is to run database with slow query tracing mode always enabled.
+But the implementation can't detect early if the request will be slow before
+it got processed so it has to record all the tracing events before making
+a decision. Recording all the tracing events with all of its parameters during
+the request execution implies sufficient overhead. This lightweight mode or
+fast slow queries tracing mode offers a solution to this problem allowing
+low-overhead slow queries tracing.
+
+#### The solution
+
+The "fast slow query logging" is a ScyllaDB feature mode that is going to
+ease the debugging related to the long requests even further. It minimizes
+the tracing session related overhead to its minimum allowing it to be always
+enabled.
+
+In a nutshell, this mode tracks only CQL statement and related request
+parameters. It effectively omits all the tracing events during the processing.
+
+When enabled, it will work in the same way `slow query tracing` does besides
+that it will omit recording all the tracing events. So that it will not
+populate data to the `system_traces.events` table but it will populate
+trace session records for slow queries to all the rest: `system_traces.sessions`,
+`system_traces.node_slow_log`, etc.
+
+Other tracing modes work as usual with that mode enabled.
+
+#### How to enable and configure
+
+By default fast slow query logging is disabled.
+
+There is a REST API that allows configuring and querying the current
+configuration of this feature.
+
+To request current state of the tracing run:
+
+    $ curl http://<node address>:10000/storage_service/slow_query
+
+    {"enable": false, "ttl": 86400, "threshold": 500000, "fast": false}
+
+To enable fast slow query tracing run:
+
+    $ curl --request POST --header "Content-Type: application/json" --header "Accept: application/json" http://<node address>:10000/storage_service/slow_query?enable=true&fast=true
```

Comment:
The last argument was quoted for a reason :) Without it, `&` is interpreted by bash as running whatever was typed before it in a separate thread, so this snippet actually just sends `enable=true` and does it in the background, and later sets up a bash variable `fast` with value `true`. So, the solution was to actually add the finishing `"`, not removing the starting one.

Reply:
@psarna personally, I prefer escaping. Does `\&` and `\?` work for you?

Reply:
sure, I'm fine with anything as long as it's easy to copy-paste the command

Reply:
Ok, I returned the variant of having the last argument escaped with ". Now shall be fine.